core/src/com/unciv/ui/civilopedia/CivilopediaScreen.kt

```diff
@@ -315,4 +314,15 @@ class CivilopediaScreen(
             game.setScreen(CivilopediaScreen(ruleset, previousScreen, currentCategory, currentEntry))
         }
     }
+
+    companion object {
+        /** Test whether to show Religion-specific items, does not require a game to be running */
+        // Here we decide whether to show Religion in Civilopedia from Main Menu (no gameInfo loaded)
+        fun showReligionInCivilopedia(ruleset: Ruleset? = null) = when {
```

Comment:
No, the scope for me was - Civilopedia out-of-game is inconsistent because the Religion flag was determined in different ways by different parts, and I wanted to centralize that. A **_sensible_** solution is a different matter - then the entire option would go and be left to Beliefs.isEmpty => ruleset alone. But that is not my decision to make.

Reply:
Yes, the legacy "getDescription" functions are preparing UI text, and lived in model long before my time, so the getCivilopediaText pendants naturally were placed alongside. I've repeatedly thought they should move, but never found enough motivation.